singlevenue/src/main/java/com/mappedin/examples/singlevenue/LocationActivity.java

```diff
@@ -124,6 +127,16 @@ public void run() {
             }
         });
     }
+    void showMap(){
```

Comment:
Not sure if this function is necessary. Is there a reason we need the runOnUiThread here? I tried removing it and things seemed to still be working.